### PR TITLE
feat: cocktail card - save to image

### PR DESCRIPTION
**Pull request by mildrrnt**
## Change made

- [ ] Bug fixes
- [x] New features
- [ ] Breaking changes

## Describe what you have done

- Add 6 pages for cocktail cards with save to image button, allowing users to capture result without button components on screen.

### New Features

- Save to image

### CODE REVIEW

**Comment by shalluv** [comment]
You can run `pnpm format` to format all files.

File: src/components/story/ButtonSet.tsx [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+  };
+
+  const handleClick = () => {
+    location.href = "../";
+  };
+
+  return (
+    <div
+      id="buttonSet"
+      className="relative mx-auto my-auto mt-2 flex w-[16rem] gap-4"
```
Comment (shalluv) [inline src/components/story/ButtonSet.tsx]: Please avoid arbitrary value
```suggestion
      className="relative mx-auto my-auto mt-2 flex w-64 gap-4"

```

**Comment by mildrrnt** [comment]
> You can run `pnpm format` to format all files.  @shalluv 

Do you have any suggestion on formatting? I think what cause the issue is the text inside 

Before running `pnpm format`, it looks like this:

image 

However, after running `pnpm format`, it looks like this and affect the formatting of the actual page.

image

**Comment by shalluv** [comment]
It's actually not a great idea to separate lines like that. Please use `<br />` tag and everything will be fine!

**Comment by mildrrnt** [comment]
Thank you for your comment kub. Sorry for my frequent commits and pushes, I run `pnpm format` twice/thrice for it to pass.

Another thing is that I just realized that I could run `pnpm lint` locally...

File: src/components/story/FooterDecorator.tsx [hunk 2]
```diff
@@ -0,0 +1,44 @@
+const FooterDecorator = () => {
+  return (
+    <div className="absolute inset-x-0 -left-[1.25rem] bottom-32 opacity-80">
+      <img
+        className="absolute left-0 top-[48px] h-32 w-[152px]"
+        src="/src/images/elements/calendar.png"
```
Comment (shalluv) [inline src/components/story/FooterDecorator.tsx]: Please import it first as 
```tsx
import Calendar from "@/assets/images/elements/calendar.png"

// then

<img ... src={Calendar.src} ... />
```

**Comment by shalluv** [comment]
Since Card pages have the same layout, is it possible to create a shared layout or shared components for them?

**Comment by shalluv** [comment]
Furthermore, the assets are already located in the `src/assets` directory on the `dev` branch. You can merge (`git pull origin dev`) the `dev` branch into this one to access them. Please remove any redundant images from the `src/images` directory.

**Review by shalluv — APPROVED** [review]
Great job!

**Pull request merged**